lib/mcb/commands/providers/mark_providers_as_scitts.rb

```diff
@@ -0,0 +1,62 @@
+    "Lampton LWA SCITT", "Landau Forte College Derby SCITT", "Leeds SCITT", "Leicester & Leicestershire SCITT",
+    "Leicestershire Secondary SCITT", "Lincolnshire Teaching School Alliance SCITT", "Manchester Nexus SCITT", "Mersey Boroughs ITT Partnership",
+    "Mid Essex Initial Teacher Training", "Mid Somerset Consortium for Teacher Training", "NELTA (North East London Teaching Alliance)", "Norfolk Teacher Training Centre",
+    "North East Partnership SCITT (Physical Education)", "North Essex Teacher Training (NETT)", "North Lincolnshire SCITT Partnership", "North Manchester ITT Partnership",
+    "North Tyneside SCITT", "North West SHARES SCITT", "North Wiltshire SCITT", "Northampton Teacher Training Partnership",
+    "Northern Lights SCITT", "Nottinghamshire Torch SCITT", "Partnership London SCITT (PLS)", "Pennine Lancashire SCITT",
+    "Poole SCITT", "Portsmouth Primary SCITT", "Prestolee SCITT", "Primary Catholic Partnership SCITT",
+    "Prince Henry's High School & South Worcestershire SCITT", "Red Kite Teacher Training", "Redcar and Cleveland Teacher Training Partnership", "Ripley TSA SCITT",
+    "Royal Academy of Dance", "Sacred Heart Newcastle SCITT", "SAF Initial Teacher Training", "SCITT in East London Schools (SCITTELS)",
+    "Shotton Hall SCITT", "Somerset SCITT Consortium", "South Birmingham SCITT", "South Coast SCITT",
+    "South West Teacher Training", "St. George's Academy Partnership", "St. Joseph's College Stoke Secondary Partnership", "Star Teachers SCITT",
+    "Stockton-on-Tees Teacher Training Partnership", "Stourport SCITT", "Suffolk and Norfolk Primary SCITT", "Sutton Park SCITT",
+    "Sutton SCITT", "Teach East", "Teach Kent & Sussex", "Teach SouthEast",
+    "Thamesmead SCITT", "The Basingstoke Alliance SCITT", "The Bedfordshire Schoolsâ Training Partnership", "The Cambridge Partnership",
+    "The Deepings SCITT", "The Grand Union Training Partnership", "The Hampshire LEARN SCITT Partnership", "The Havering Teacher Training Partnership",
+    "The John Taylor SCITT", "The Learning Institute South West", "The National Mathematics and Physics SCITT", "The National Modern Languages SCITT",
+    "The OAKS (Ormiston and Keele SCITT)", "The Pimlico-London SCITT", "The Sheffield SCITT", "The Shire Foundation",
+    "The Solent SCITT", "The South Downs SCITT", "The Tommy Flowers SCITT Milton Keynes", "Three Counties Alliance SCITT",
+    "Titan Partnership Ltd", "TKAT SCITT", "Tudor Grange SCITT", "Two Mile Ash ITT Partnership",
+    "Wandsworth Primary Schoolsâ Consortium", "Wessex Schools Training Partnership", "West Midlands Consortium", "Wildern Partnership",
+    "Yorkshire and Humber Teacher Training", "Yorkshire Wolds Teacher Training"
+  ]
+
+  puts 'Updating SCITTS for 2019'
+  MCB.get_recruitment_cycle('recruitment-year': 2019).providers.where(provider_name: scitts).each do |provider|
```

Comment:
`MCB.get_recruitment_cycle` looks to me like it's a utility method to help parse cmdline args ... so unless you're passing `opts` to it to see what recruitment cycle the user wanted (defaulting to the current one), I would've just gone with a `RecruitmentCycle.each` loop or something. :)

Reply:
:man_facepalming: I just completely forgot about ActiveRecord when i wrote that apparently